modules/mocked_authentication/app/controllers/mocked_authentication/credential_controller.rb

```diff
@@ -18,6 +20,17 @@ def authorize
       render json: { errors: e }, status: :bad_request
     end
 
+    def credential_list
+      type = params[:type].presence
+
+      validate_index_params(type)
+      mock_profiles = Mockdata::Reader.find_credentials(credential_type: type)
+
+      render json: { mock_profiles: }
```

Comment:
I've gotta admit, seeing the hashes like this after the Ruby 3.1 upgrade still catches me off guard for a second before I realize this is how it's supposed to be, ha

Reply:
I'm starting to get used to it surprisingly quickly! It really does cut a lot of unnecessary text out.